src/scss/menulog.scss

```diff
@@ -0,0 +1,87 @@
+// =================================
+// Menulog Color palette
+//
+// Only colour overrides should be listed here
+// =================================
+
+$green                        : #22aa1e;
+$green--dark                  : #148212;
+
+$green--darkest               : #11530f;
+
+$orange                       : #c25700;
+
+// Brand specific colours
+// Not accessible in all circumstances but available if needed in certain designs
+$brand--green                 : #26bf22;
+$brand--green--light          : #52e04e;
+$brand--orange                : #ff5f37;
+$brand--orange--dark          : #f55f38; // search button hover bg colour
+$brand--orange--darker        : #ea4a1d; // search button active bg colour
+
+
+// =================================
+// Color Scheme
+// =================================
+// These are the variables that should be used directly in our SCSS (globally)
+
+$color-secondary        : $green--dark;
+$color-link-default     : $color-secondary;
+$color-link-visited     : $color-secondary;
+$color-link-hover       : $brand--green;
+$color-link-active      : $green--darkest;
+$color-focus-outline    : $green;
+
+
+// Overides - theme overide colours for specific components
+// Button Colours
+$btn-primary-bgColor                  : $green;
+$btn-primary-bgColor--highlight       : $brand--green;
+$btn-primary-bgColor--focus           : $green--dark;
+
+$btn-secondary-bgColor                : transparent;
+$btn-secondary-bgColor--highlight     : green--offWhite;
+$btn-secondary-bgColor--active        : green--offWhite;
+$btn-secondary-textColor              : $green;
+
+$btn-tertiary-bgColor                 : transparent;
+$btn-tertiary-bgColor--highlight      : green--offWhite;
+$btn-tertiary-bgColor--active         : $white;
+$btn-tertiary-textColor               : $grey--dark;
+
+$btn-outline-borderColor              : $brand--green;
+$btn-outline-borderColor--hover       : $brand--green;
+$btn-outline-borderColor--active      : $green--dark;
+$btn-outline-bgColor--hover           : green--offWhite;
+$btn-outline-textColor                : $green;
+$btn-outline-textColor--hover         : $brand--green;
+$btn-outline-textColor--active        : $green--dark;
+
+$btnToggle-textColor                  : $grey--dark;
+$btnToggle-textColor--active          : $green;
+$btnToggle-border--active             : $green;
+
+$btn-roundedOutline-fillColor         : $green;
+$btn-roundedOutline-fillColor--active : $green--dark;
+$btn-rounded-fillColor                : $green;
+
+// Component Colours
```

Comment:
Can you put a comment above this line – it would be best to have the component colours as part of the component itself rather than as part of the colour palette (as that helps tidyup if a component gets deleted).

The way I'd think this would work would be to have a variable set in the Sass file to set the theme that you want and then the component will set the variables up according to that theme.

So something like `$fozzie-theme: 'menulog'` inside the menulog dependencies Sass file will then switch a load of colours to green in the component.

If you can add a comment in just to tidy this up to use a fozzie variable for these components, I'll create an issue to look at this when we tidy this up and move the styling fully into Fozzie (and I'll hopefully remember what I was thinking now 😉)